src/app/legacy/psammead/psammead-styles/src/index.stories.jsx

```diff
@@ -80,7 +79,7 @@ const getFontStyles = fontName => {
     .replace(/: /g, ':"')
     .replace(/;/g, '",');
 
-  const { fontFamily, fontWeight } = json5.parse(camelCase(fontStyle));
+  const { fontFamily, fontWeight } = JSON.parse(camelCase(fontStyle));
```

Comment:
Unsure if this will work, but Psammead isn't hooked up to Storybook anyway, so I think we should just fix this if/when we do that work.